### ISSUE TITLE
publish December, 2016 column

### ISSUE THREAD

**Issue by whit537** [whit537/openorg#7]
Following on from https://github.com/gratipay/inside.gratipay.com/issues/757#issuecomment-256234457, I want to write about something practical! I'm thinking of collecting examples of open company handbooks—GitLab, Enspiral, Loomio, Gratipay ...

**Comment by semioticrobotic** [comment]
Amazing idea! That'd be a valuable resource, for sure, and I'm sure you'd bring a keen and analytical eye to potential comparisons.

**Comment by whit537** [comment]
@semioticrobotic I think the handbooks idea is sound, and that a follow-up to the "free rider" [post](https://opensource.com/open-organization/16/11/open-source-free-rider-problem) is probably in order, given the interest and awesome feedback it's generated. That said, I wanted to ask you about the possibility of cross-posting something I just posted on the Gratipay blog (which has been a bit neglected since I started writing for OpenSource.com, to be honest! :). The piece is "[Seven Ways to Crowdfund Your Open Source Project](https://gratipay.news/seven-ways-to-crowdfund-your-open-source-project-f169b42311b2)." Would this be appropriate to use for my OpenSource.com column? What are your policies around cross-posting?

Hope you had a great Thanksgiving with your family and enjoy the long weekend as well! 🍗  :-)

**Comment by semioticrobotic** [comment]
@whit537 Love the post (very useful resource!). We do not often mind reposting already existing content, but in almost all cases prefer to publish something that is at least slightly re-written for Opensource.com (quite simply: in our experience, reposting content with little or no alteration doesn't enhance the visibility of either the original _or_ the repost, thanks to the vicissitudes of PageRank). One option we might consider would involve putting a version of this particular post in the queue, then working together on something else as your "official" column for December or early January. Your call, in the end!

**Comment by whit537** [comment]
Gotcha. I was hoping to get a BOGO here on Black Friday, but I guess there's no such thing as a free lunch. Or something. :-)

Let's keep the "Seven Ways" post out of the OpenSource.com queue for now. I'll look for a chance to draft another post. Sounds like the queue is still pretty full, ya?

**Comment by semioticrobotic** [comment]
@whit537 Indeed; the queue is still fairly healthy. If you want to use December to draft a column for January, we'd be fine. Again: completely your call.

**Comment by whit537** [comment]
[Gone fishin'](https://twitter.com/whit537/status/816770410830266369).

**Comment by whit537** [comment]
Started a [draft](https://docs.google.com/document/d/1DrAmDHyDUZycdA4bS56CvOQylsqF_A2vHmSxaDxCBYg/edit?usp=sharing), currently collecting handbooks and reading them.

**Comment by whit537** [comment]
Some good leads at https://handbook.enspiral.com/#other-handbooks ...

**Comment by whit537** [comment]
Gosh, this is a going to be a lot to digest. :-)

**Comment by whit537** [comment]
[Done](https://cse.google.com/cse/publicurl?cx=013947747009183698677:wpxlyhpipjc)! 💃 

![screen shot 2017-01-05 at 10 22 43 am](https://cloud.githubusercontent.com/assets/134455/21685790/eeaacc08-d330-11e6-8b63-943666b4339a.png)

**Comment by whit537** [comment]
https://twitter.com/whit537/status/817031862593998848

**Comment by whit537** [comment]
https://github.com/basecamp/handbook

**Comment by whit537** [comment]
In honor of DHH's first [appearance](https://opensource.com/open-organization/17/8/basecamp-transparent-pay) on the openorg channel, I've added Basecamp to the search engine!
